cfgov/v1/models/base.py

```diff
@@ -197,7 +193,7 @@ def match_all_topic_tags(queryset, tags):
             if and_filtering:
                 # By default, we need to match at least one tag
                 # If specified in the admin, change this to match ALL tags
-                related_queryset = match_all_topic_tags(related_queryset, tags)
+                related_queryset = related_queryset.filter(tags__name__in=tags)
```

Comment:
This is awesome! One quick thought is that, since we're updating the logic to `match_all_topic_tags` (a function I recently introduced to make it clearer what this code does), would it make more sense to update that function rather than remove it?  Just a thought but I don't feel strongly. It's mostly for readability purposes.

Reply:
Once I had that function down to one line, it didn't seem worthwhile to keep it?

It turns out, I actually broke the logic a bit, so this PR should now be considered on hold.

Reply:
Yeah -- I don't think the number of lines should necessarily dictate when something is a function or not. Some discussion here: https://softwareengineering.stackexchange.com/questions/107669/one-line-functions-that-are-called-only-once